include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -599,6 +629,11 @@ class FEPointEvaluation
   Point<dim>
   unit_point(const unsigned int point_index) const;
 
+  /**
+   * Number of quadrature points of the current cell/face.
+   */
+  unsigned int n_q_points;
```

Comment:
Can we make this variable `const` and use `const_cast<unsigned int&>(n_q_points) = ...` in lines 896 and similar below? I would really like this variable to be seen as `const` to the outside, in analogy to https://github.com/dealii/dealii/blob/7331b14d3e3d68033c9dc0b64631ce3b66db7bc2/include/deal.II/fe/fe_values.h#L2433 or https://github.com/dealii/dealii/blob/7331b14d3e3d68033c9dc0b64631ce3b66db7bc2/include/deal.II/matrix_free/fe_evaluation.h#L2351
Since all these calls happen inside a non-static member function of the class, I do not think we can trigger undefined behavior when a compiler would completely eliminate the `const` variable without having a storage location for it (in which case `static_cast<unsigned int&>` would fail with a compile error).